src/admin-views/aggregator/tabs/import-form.php

```diff
@@ -1,5 +1,6 @@
 <?php
-$has_license_key = Tribe__Events__Aggregator::is_service_active();
+$has_license_key = tribe( 'events-aggregator.main' )->is_service_active();
```

Comment:
Since it is the return of a function it can be called non statically. If I tried to call it statically I would get a `T_PAAMAYIM_NEKUDOTAYIM` error in PHP 5.2. I just tested this to double check. Should work in new and old version of PHP, and we're using the same line of code in a few other places.